### ISSUE TITLE
Styling the "Ads" Button

### ISSUE THREAD

**Issue by matteioo** [x0a/uBO-YouTube#5]
I would appreciate if the Ads button would be styled in a more intuitive way. For example:
- with a [toggle switch](https://codepen.io/matteio/pen/QmrORK)
- with a [primary](https://material-components.github.io/material-components-web-catalog/#/component/button?label=Ads&type=raised) or [secondary](https://material-components.github.io/material-components-web-catalog/#/component/button?label=Ads&type=outlined) button or a [button](https://material-components.github.io/material-components-web-catalog/#/component/button?label=Ads&type=unelevated) adopting googles Material Design

  - to visualize it is cheched or not you can use an Icon from Fontawesome which are free to use.

**Comment by matteioo** [comment]
May it be possible to use the style form the Youtube page and recolor the button?

**Comment by x0a** [comment]
Great suggestions.
I think your second idea would work better because it's not too disruptive to the look of the bottom bar; new users should be able to tell instantly what the button is doing. Adding text next to a toggle might look a little off given the space constraints.

YouTube used to expose discrete CSS classes that you could use to seamlessly integrate into the page, but the new Polymer design heavily relies on CustomElements and instead of using the Material CSS directly, they compile what they need into classes specific to the buttons they use. (ie ytd-subscribe-button-renderer.ytd-video-secondary-info-renderer). 

So basically I need to recreate these buttons with the Material CSS. I'll get back to you with a test build if you're interested in trying it.

**Comment by x0a** [comment]
![greencheck-0](https://user-images.githubusercontent.com/13019784/68242358-fbda1080-ffd5-11e9-98a4-ea23bae6595d.png)
![greencheck-1](https://user-images.githubusercontent.com/13019784/68242360-fbda1080-ffd5-11e9-8278-df8c8ae0da21.png)
![greencheck-2](https://user-images.githubusercontent.com/13019784/68242361-fbda1080-ffd5-11e9-8686-64f5cd66019e.png)




Reduced opacity when button is "off", invert color, add checkmark, and increase opacity when button is "on". Is that a bit more intuitive?

**Comment by matteioo** [comment]
This looks already much better. With my first idea I wanted to show that of course there are really intuitive possibilities but the costs were the ui integration. Even though the look is better than the current solution. 

Your latest screenshots are absolutely an eye catcher! How about putting a cross if ads disabled? Can't wait seeing it on the youtube page. AWESOME

**Comment by matteioo** [comment]
But removing the icon would also be a possibility. This is all about testing 👍

**Comment by contrabyte** [comment]
Totally agree that the button needed restyling and I would much prefer the new look! I would be a little concerned about adding an X icon because it might make users think that clicking the button would block ads instead of whitelisting. Just my 2 cents

**Comment by x0a** [comment]
Thanks for the input! Should be available in 0.9.6, maybe in a day or so. I 100% welcome any further suggestions, so if you have any other suggestions for the 'Ads' button then feel free to re-open the issue or open a new issue for anything else.
